GNUmakefile

```diff
@@ -145,7 +147,8 @@ dev: dev-build
 
 dev-build:
 	mkdir -p bin
-	CGO_ENABLED=0 go build -o ./bin -ldflags "$(GOLDFLAGS)" -tags "$(GOTAGS)"
+	CGO_ENABLED=0 go install -ldflags "$(GOLDFLAGS)" -tags "$(GOTAGS)"
+	cp -f ${MAIN_GOPATH}/bin/consul ./bin/consul
```

Comment:
I am still weary about installing things to `PATH` from a target called `build`, but this was the previous behaviour, and it seems you feel strongly about preserving it, so I guess I will have to live with it.